app/models/ReferenceDataPayload.scala

```diff
@@ -54,5 +54,5 @@ class ReferenceDataListsPayload(data: JsObject) extends ReferenceDataPayload {
 }
 
 object ReferenceDataListsPayload {
-  def apply(v1: JsObject): ReferenceDataListsPayload = new ReferenceDataListsPayload(v1)
+  def apply(v2: JsObject): ReferenceDataListsPayload = new ReferenceDataListsPayload(v2)
```

Comment:
Could this val be given a more appropriate name? e.g. `data` like in the class definition